### DISCUSSION TITLE
Knex and Postgis with Lucid not working: no existe la función st_geomfromtext(unknown, unknown)

### DISCUSSION THREAD

**Discussion by fabricioAburto** [adonisjs/core#3883] [General]
## Prerequisites
Hi guys, I'm traying save a location using postgress (lucid knex and postgis) and following docs carefully but I think the binding process is failling in your site. 

```cmd
❯ error :
  insert into "users" ("created_at", "email", "fullname", "id", "location", "password", "picture", "range_distance", "remember_me_token", "status", "updated_at", "username") values ($1, $2, $3, $4, ST_geomFromText($5, $6), $7, $8, $9, $10, $11, $12, $13) returning "id" - no existe la función st_geomfromtext(unknown, unknown)
```

As you can see there is `unknown` into the `st_geomfromtext` function arguments and I think this is causing this issue: 

```
No function matches on the name and types of arguments. It may be necessary to add explicit type casting.
```

[Oficial Docs](https://docs.adonisjs.com/cookbooks/using-knex-postgis-with-lucid) but nothing.

## Package version
    "@adonisjs/core": "^5.8.6",
    "@adonisjs/lucid": "^18.1.0",
    "knex-postgis": "^0.14.3"

## Node.js and npm version
v1613.2 - 8.3.0 

## Sample Code (to reproduce the issue)
same as you put in the oficial docs.

```ts
class Point {
  @column({
    prepare: (value?: string) => {
      return value ? Database.st().geomFromText(value, 4326) : value
    }
  })
  public geom: string
}

const point = new Point()
point.geom = 'Point(0 0)'
await point.save()
```
[Following step by step carefully](https://docs.adonisjs.com/cookbooks/using-knex-postgis-with-lucid) but nothing.

**Comment by Dyoma3** [comment]
Could you share your project or just a new project with your postgis logic?
